identity-apps-core/apps/recovery-portal/src/main/resources/org/wso2/carbon/identity/mgt/recovery/endpoint/i18n/Resources_pt_PT.properties

```diff
@@ -271,6 +275,9 @@ enter.your.username=Entre com seu nome de usuário
 error.enter.code=Por favor, digite o código!
 choose.other.option=Escolha uma opção diferente
 resend.code.success=Um novo OTP foi enviado para o seu telemóvel.
+enter.code.sent.smsotp=Insira o código enviado para o seu telemóvel.
+sms.otp.resend.code=Reenviar código.
+sms.otp.submit.button=Continuar.
```

Comment:
Shall we remove the `.` from button text?

Reply:
addressed with https://github.com/wso2/identity-apps/pull/6418/commits/34b1b04cadee820b1b64ff882b6c969a25b8d4df